documentation/en/chain-indexer-overview-for-operators.md

```diff
@@ -0,0 +1,415 @@
+
+1. **Review the [preparation steps](#preparation)**
+   - The disk space and upgrade times will be much smaller than the ["upgrade with existing chain state" path](#upgrade-when-using-existing-lotus_path-chain-state) assuming this is a non-archival node that is only indexing a limited number of days of epochs.
+2. **Ensure the node is stopped and won't take any traffic initially upon starting**
+   - Example: prevent a load balancer from routing traffic to the node.
+3. **Update Configuration**
+   - Modify the Lotus configuration to enable the `ChainIndexer` as described in the [`ChainIndexer Config` section above](#chainindexer-config). 
+4. **Start lotus with the snapshot import**
+   - `lotus daemon --import-snapshot`
+5. **Wait for the Lotus daemon to sync**
+   - As the Lotus daemon syncs the chain, the ChainIndexer will automatically index the synced messages, but it will not automatically sync ETH RPC events and transactions.
+6. **Backfill so ETH RPC events and transactions are indexed as well**
+   - See the ["Backfill" section below](#backfill).
+   - This will look something like `lotus-shed chainindex validate-backfill --from <head_epoch> --to <epoch_corresponding_with_how_much_state_in_past_want_to_index>  --backfill`
+     - Example: if the current head is epoch 4360000 and one wants to index a day's worth of epochs (2880), then they'd use `--from 4360000 --to 4357120`
+6. **Ensure node health**
+   - Perform whatever steps are usually done to validate a node's health before handling traffic (e.g., log scans, smoke tests)
+7. **Route traffic to the backfilled node that is now using ChainIndexer**
+
+## Backfill
+There is no automated migration from [pre-ChainIndexer indices](#previous-indexing-system) to the [ChainIndex](#chainindexer-indexing-system).  Instead one needs to index historical chain state (i.e., backfill), if RPC access to that historical state is required. (If curious, [read why](#wny-isnt-there-an-automated-migration-from-the-previous-indexing-system-to-the-chainindexer-indexing-system).)
+
+### Backfill Timing
+
+Backfilling the new `ChainIndexer` was [benchmarked to take approximately ~12 hours per month of epochs on a sample archival node doing no other work](https://github.com/filecoin-project/lotus/issues/12453#issuecomment-2405306468). Your results will vary depending on hardware, network, and competing processes.  This means if one is upgrading a FEVM archival node, they should plan on the node being out of production service for ~10 days.  Additional nodes to update don't need to go throuh the same time-intensive process though.  They can get a `${LOTUS_PATH}/chainindex/chainindex.db` copied from a trusted node per the [upgrade steps](#upgrade).  
+
+### Backfill Disk Space Requirements
+
+As of 202410, ChainIndexer will accumulate approximately ~340 MiB per day of data, or 10 GiB per month (see [here](https://github.com/filecoin-project/lotus/issues/12453)).
```

Comment:
This is correct. See https://filecoinproject.slack.com/archives/CP50PPW2X/p1729413621133599.